OneCommunityGlobal/HighestGoodNetworkApp
# Shereen- Equipment List View Component

Created Equipment List View Component
Created URL for equipment List View
Created actions for Equipment List View

# Description
Created components for Equipment list view
Implements # (WBS) 9.1.1

## Related PRS (if any):
This frontend PR is related to the (https://github.com/OneCommunityGlobal/HGNRest/pull/992) backend PR.
To test this frontend PR you need to checkout the (https://github.com/OneCommunityGlobal/HGNRest/pull/992) backend PR.
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. login as any user
5. Go to /bmdashboard (http://localhost:3000/bmdashboard) and log in again.
6. Navigate to [/bmdashboard/equipment] (http://localhost:3000/bmdashboard/equipment)
7. Verify all the table functionalities like open/close modals, sorting, and filtering for equipment view is working


## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/7041610/914f2a4d-b04d-48ac-b91d-0f6fdec93b15
